pom.xml

```diff
@@ -28,30 +28,30 @@
 
     <properties>
         <!-- Versions for required dependencies -->
-        <guava.version>30.0-jre</guava.version>
-        <kiwi.version>0.18.0</kiwi.version>
+        <guava.version>30.1-jre</guava.version>
+        <kiwi.version>0.19.0</kiwi.version>
 
         <!-- Versions for provided dependencies -->
         <byte.buddy.version>1.10.18</byte.buddy.version>
         <commons-text.version>1.9</commons-text.version>
         <curator.version>2.13.0</curator.version>
-        <dropwizard.version>2.0.16</dropwizard.version>
-        <dropwizard.metrics.version>4.1.16</dropwizard.metrics.version>
+        <dropwizard.version>2.0.19</dropwizard.version>
+        <dropwizard.metrics.version>4.1.17</dropwizard.metrics.version>
         <dropwizard.modules.dropwizard-jdbi.version>2.0.12</dropwizard.modules.dropwizard-jdbi.version>
         <embedded-postgres.version>1.2.6</embedded-postgres.version>
         <h2.version>1.4.200</h2.version>
         <hibernate.version>5.4.24.Final</hibernate.version>
         <hibernate-validator.version>6.1.6.Final</hibernate-validator.version>
         <jakarta.annotation-api.version>1.3.5</jakarta.annotation-api.version>
         <javassist.version>3.27.0-GA</javassist.version>
-        <jackson.version>2.10.5</jackson.version>
+        <jackson.version>2.10.5.1</jackson.version>
```

Comment:
Do we want to use the range like in kiwi?

Reply:
Only need to do this when necessary.  I think we should be as explicit as possible.